scripts/run_lighthouse_tests.py

```diff
@@ -106,13 +116,18 @@ def run_lighthouse_puppeteer_script(record: bool = False) -> None:
     stdout, stderr = process.communicate()
     if process.returncode == 0:
         print(stdout)
+        entities: dict[str, str] = {}
```

Comment:
Maybe add a comment to explain the structure of this, and also explain what an "entity" is (perhaps with examples).

Reply:
Done